src/ontology/analysis/analyze_mondo_stats/README.md

```diff
@@ -0,0 +1,307 @@
+
+- Query Mondo for a _list_ of all Mondo classes that are in the human disease branch, i.e. children of MONDO:0700096 'human disease' and have an xref to NCIT. Run from `mondo/src/ontology/analysis/analyze_mondo_stats` as: 
+```
+robot reason -i ../../mondo-edit.obo query --use-graphs true -q sparql/get_mondo_class_list_with_ncit_xref.ru data/input/all_mondo_classes_with_ncit_xref.tsv
+```
+
+- Check for overlapping NCIT CURIEs in each file by running these commands from the Terminal:
+```
+$ awk -F'\t' 'NR==FNR {xref[$1]; next} $3 in xref' data/ncit_neoplasm_children.tsv data/input/all_mondo_classes_with_ncit_xref.tsv > data/all_mondo_classes_with_ncit_xref_neoplasm-branch.tsv
+
+$ wc data/all_mondo_classes_with_ncit_xref_neoplasm-branch.tsv
+    3804   20177  229560 data/all_mondo_classes_with_ncit_xref_neoplasm-branch.tsv
+```
+
+#### Results for Alignment with the source - Total number of diseases in sources
+- DOID	11390
+- ICD10CM	9142
+- NCIT	3804
+- OMIM	9333
+- Orphanet	11035
+- UMLS	20738
+
+
+### Alignment with the source - Number of terms from the sources that are in Mondo (as Mondo:equivalent)
+To generate these counts, follow the steps above for "Alignment with the source - Total number of diseases in sources", but create new queries where the xref must also have a source annotation of `MONDO:equivalentTo`.
+
+- Run from `mondo/src/ontology/analysis/analyze_mondo_stats` as: 
+```
+robot reason -i ../../mondo-edit.obo query --use-graphs true -q sparql/get_all_human_disease_mondo_classes_with_xref-equivalentTo.ru data/output/all_human_disease_mondo_class_xref_count-equivalentTo.tsv
+```
+
+
+#### Alignment to the source NCIT - Number of terms from the sources that are in Mondo (as Mondo:equivalent)
+- Query Mondo for a _list_ of all Mondo classes that are in the human disease branch, i.e. children of MONDO:0700096 'human disease' and have an xref to NCIT and also have an xref source annotation of `MONDO:equivalentTo`. Run from `mondo/src/ontology/analysis/analyze_mondo_stats` as: 
+```
+robot reason -i ../../mondo-edit.obo query --use-graphs true -q sparql/get_mondo_class_list_with_ncit_xref-equivalentTo.ru data/input/all_mondo_classes_with_ncit_xref-equivalentTo.tsv
+```
+
+- Check of overlapping NCIT CURIEs in each file by running these commands from the Terminal:
+```
+$ awk -F'\t' 'NR==FNR {xref[$1]; next} $3 in xref' data/input/ncit_neoplasm_children.tsv data/input/all_mondo_classes_with_ncit_xref-equivalentTo.tsv > data/output/all_mondo_classes_with_ncit_xref_neoplasm-branch-equivalentTo.tsv
+
+$ wc data/output/all_mondo_classes_with_ncit_xref_neoplasm-branch-equivalentTo.tsv
+    3452   18507  210236 data/output/all_mondo_classes_with_ncit_xref_neoplasm-branch-equivalentTo.tsv
+```
+
+#### Results for Alignment with the source - Number of terms from the sources that are in Mondo (as Mondo:equivalent)
+- DOID	11185
+- ICD10CM	1116
+- NCIT	3452
+- OMIM	9141
+- Orphanet	8176
```

Comment:
For orphanet, we should limit our analysis to the orphanet disease and disease subtypes.
I don't think this is the case here (based on how many are not reported to be in Mondo)

Reply:
I added another query for Orphanet to get the count limited to only `ordo_disorder` and `ordo_subtype_of_a_disorder` subsets. I am not sure what you mean by "I don't think this is the case here (based on how many are not reported to be in Mondo)".